scripts/provision.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -exuo pipefail
+
+# Install packages
+sudo dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+sudo dnf install -y --nogpgcheck osbuild-composer composer-cli git curl jq expect wget ansible podman httpd qemu-img qemu-kvm virt-install libvirt-client libvirt-daemon-kvm bash-completion
+sudo rpm -qa|grep osbuild
+
+# Prepare osbuild-composer repository file
+sudo mkdir -p /etc/osbuild-composer/repositories
+sudo cp osbuild-composer/test/data/repositories/rhel-84.json /etc/osbuild-composer/repositories/rhel-8-beta.json
+
+# Prepare RSA key files(this step is not needed in jenkins run env, keep it here for local test.)
+#sudo mkdir -p /usr/share/tests/osbuild-composer/keyring
```

Comment:
Could you please squash your commits after you fix this issue. This PR is just for add one test, do not need more commits. Thanks.

Reply:
sure, I will.

And for the key files, in your previous comment, you said it is duplicated with jenkins file, so I commented it out.

Is the key files still needed here?

Reply:
My [comment](https://github.com/virt-s1/rhel-edge/pull/6#discussion_r607554033) about duplicated is about copy provision.sh, not copy key files. YES, the key file must be copied, because test needs key file. There's no any key file related commands in Jenkinsfile.

Reply:
ok, fixed.